### PR TITLE
Feature/35332 new govuk pupil register

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
Pages updates:
- [x] Pupil register
- [x] Add pupil
- [x] Edit pupil
- [x] Add batch pupils

New component support:
- [x] Breadcrumbs

### CODE REVIEW

File: admin/assets/javascripts/gds-pupil-form.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * Pupil access arrangements selection
+ */
+
+if (!window.MTCAdmin) {
+  window.MTCAdmin = {}
+}
+
+/* global $ */
+(function () {
+  'use strict'
```
Comment (GuyHarwood) [inline admin/assets/javascripts/gds-pupil-form.js]: intentionally not at top of file?

File: admin/assets/javascripts/gds-pupil-form.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/**
```
Comment (GuyHarwood) [inline admin/assets/javascripts/gds-pupil-form.js]: where was this code defined/kept previously?

File: admin/controllers/pupil.js [hunk 3]
```diff
@@ -100,7 +101,8 @@ const getAddMultiplePupils = async (req, res, next) => {
     csvErrorFileSize = await uploadedFileService.getAzureBlobFileSize(csvErrorFile)
     req.breadcrumbs('Pupil register', '/pupil-register/pupils-list')
     req.breadcrumbs(res.locals.pageTitle)
-    res.render('school/add-multiple-pupils', {
+    res.render('pupil-register/add-multiple-pupils', {
```
Comment (GuyHarwood) [inline admin/controllers/pupil.js]: 🎖

File: admin/assets/javascripts/gds-pupil-form.js (hunk 1 above)
Comment (GeorgeChatzigiannis) [reply admin/assets/javascripts/gds-pupil-form.js]: @GuyHarwood nope was a mistake, it's now moved on top for all the files I have introduced.

File: admin/assets/javascripts/gds-pupil-form.js (hunk 2 above)
Comment (GeorgeChatzigiannis) [reply admin/assets/javascripts/gds-pupil-form.js]: @GuyHarwood I reailised `pupil-form.js` is not being used elsewhere so I removed `gds-pupil-form.js`. `pupil-form.js` contains the necessary changes now.

**Comment by GeorgeChatzigiannis** [comment]
@GuyHarwood pupil form is an existing script that now has been moved under the `MTCAdmin` object.

File: admin/assets/javascripts/details.polyfill.js [hunk 4]
```diff
@@ -9,6 +9,11 @@
 ;(function () {
   'use strict'
 
+  // Prevent using this polyfill on new toolkit pages as the functionality is already provided
+  if (document.getElementsByClassName('govuk-template').length === 1) {
```
Comment (GuyHarwood) [inline admin/assets/javascripts/details.polyfill.js]: can we guarantee that there will only ever be one item with this class name?

Comment (GeorgeChatzigiannis) [reply admin/assets/javascripts/details.polyfill.js]: it's a unique class that's on the very top of the newer toolkit's layout https://design-system.service.gov.uk/styles/page-template/

**Pull request merged**